src/docs/release_strategy.md

```diff
@@ -78,13 +78,13 @@ are suffixed with `rcN`. For instance: "the `1.1.x` `stable` branch",
 
 ### `dev` naming
 `dev` releases occur between `stable` branches, and are differentiated by a `devN` suffix. The pattern
-to follow is `N.N.0devN`, where `N.N` are the _next_ `major`/`minor` branch that will be created
-and N is the next sequential number starting from `0`. For instance: "the `1.1.0dev0` `dev` release".
+to follow is `N.N.0.devN`, where `N.N` are the _next_ `major`/`minor` branch that will be created
+and N is the next sequential number starting from `0`. For instance: "the `1.1.0.dev0` `dev` release".
 
 ## Examples
 
 * Leading up to the release of `2.0.0` the release manager would create a `stable` branch with
-the literal name "`2.0.x`". They would cut release candidates named `2.0.0rc0` (and so on), and
+the literal name "`2.0.x`". They would cut release candidates named `2.0.0.rc0` (and so on), and
```

Comment:
If I read this correctly, the `.` should not precede `rcN`, unlike for `devN` where it should: https://www.python.org/dev/peps/pep-0440/#public-version-identifiers

Python versioning is not exactly semver versioning.